packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -176,6 +176,44 @@ returnsSafePromise();
 </TabItem>
 </Tabs>
 
+### `allowForKnownSafePromiseReturns`
```

Comment:
If we are at the stage where naming bikeshedding is still ok, I would say that I think this is a tricky name, and difficult to differentiate visually from `allowForKnownSafePromises`. 

`allowForKnownSafeAsyncFunctions` (technically imprecise) or `allowForKnownSafePromiseFunctions` (more precise, and reuses terminology from https://typescript-eslint.io/rules/promise-function-async/) feel better to me.

If too late to think about minutiae like naming, ignore this comment 🙂

Reply:
Not too late at all. I was actually thinking about this too... `allowForKnownSafeCalls` is shorter and more distinct from the others, how about that?

Reply:
works for me!